### PR TITLE
Move queue map and held lock set to lock manager

### PR DESCRIPTION
**Pull request by saschanaz**
Split the first part from #75.

***
<a href="https://pr-preview.s3.amazonaws.com/saschanaz/web-locks/pull/90.html" title="Last updated on Nov 11, 2021, 10:38 PM UTC (e59a2ab)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/web-locks/90/e2824eb...saschanaz:e59a2ab.html" title="Last updated on Nov 11, 2021, 10:38 PM UTC (e59a2ab)">Diff</a>

### CODE REVIEW

**Comment by saschanaz** [comment]
That's a weird error that doesn't happen locally. There is no `[W3C-STATUS-URL]` in the file 🤔

**Comment by inexorabletash** [comment]
> That's a weird error that doesn't happen locally. There is no `[W3C-STATUS-URL]` in the file 🤔

A re-run seems to have fixed it. Transient Bikeshed issue?

**Comment by inexorabletash** [comment]
Thanks @saschanaz ! Now off to rebase...

File: index.bs [hunk 1]
```diff
@@ -246,7 +246,7 @@ The [=lock-concept/waiting promise=] is not named in the above code, but is stil
 Further note that if the callback is not `async` and returns a non-promise, the return value is wrapped in a promise that is immediately resolved; the lock will be released in an upcoming microtask, and the [=lock-concept/released promise=] will also resolve in a subsequent microtask.
 </aside>
 
-Each origin has a <dfn>held lock set</dfn> which is a [=/set=] of [=lock-concept|locks=].
+Each [=/lock manager=] has a <dfn for="lock manager">held lock set</dfn> which is a [=/set=] of [=lock-concept|locks=].
```
Comment (saschanaz) [inline index.bs:249]: Forgot to say this, should these "each lock manager has ..." statement be merged to the definition of lock manager?

**Pull request merged**